### PR TITLE
refactor: Replace `_.takeRight` with `array.slice`

### PR DESCRIPTION
**Pull request by jishnu-mohan**
replace _.takeRight with array.slice #7747

### CODE REVIEW

File: lib/plugins/aws/deploy/lib/cleanupS3Bucket.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ module.exports = {
       })
       .then(response => {
         const stacks = findAndGroupDeployments(response, prefix, service, stage);
-        const stacksToKeep = _.takeRight(stacks, stacksToKeepCount);
+        const stacksToKeep = stacks.slice(stacksToKeepCount ? -stacksToKeepCount : stacks.length);
```
Comment (medikoo) [inline lib/plugins/aws/deploy/lib/cleanupS3Bucket.js:27]: I think it can be just `stacks.slice(-stacksToKeepCount)`. Do I miss something?

Comment (jishnu-diagnal) [reply lib/plugins/aws/deploy/lib/cleanupS3Bucket.js:27]: @medikoo when `stacksToKeepCount` is count is `0` ,`stacks.slice(-stacksToKeepCount)` will return the original array.

Comment (medikoo) [reply lib/plugins/aws/deploy/lib/cleanupS3Bucket.js:27]: Thanks! Indeed. In that case let's maybe put it as `stacks.slice(-stacksToKeepCount || Infinity)`

Comment (jishnu-diagnal) [reply lib/plugins/aws/deploy/lib/cleanupS3Bucket.js:27]: Yes, that will be better. Thanks!

Comment (jishnu-mohan) [reply lib/plugins/aws/deploy/lib/cleanupS3Bucket.js]: Updated PR with the suggestion from @medikoo

**Pull request merged**